src/app/examples/crud/components/crud.component.ts

```diff
@@ -29,66 +24,79 @@ import {
 })
 export class CrudComponent implements OnInit {
   private unsubscribe$: Subject<void> = new Subject<void>();
-
   routeAnimationsElements = ROUTE_ANIMATIONS_ELEMENTS;
 
-  bookFormGroup = this.fb.group(
-    this.createBook(new Date().getUTCMilliseconds().toString())
-  );
+  bookFormGroup = this.fb.group(CrudComponent.createBook());
   books: Observable<Book[]>;
```

Comment:
Nitpick, since we use the `$` syntax in the other examples, I would also do it here.